Example/Example/SeriesViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ComicsViewController.swift
+//  Example
+//
+//  Created by Davide Mendolia on 19/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import BothamUI
+
+class SeriesViewController: ExampleViewController, BothamTableViewController, SeriesUI, UITableViewDelegate {
```

Comment:
Or the UITableViewDelegate is injected or is implemented by the SeriesViewController.

I want to navigate to a detail of a series you will need to use a BothamTableViewNavigationDelegate injected as delegate